package.json

```diff
@@ -52,8 +54,7 @@
     "trailingComma": "all"
   },
   "publishConfig": {
-    "access": "public",
-    "directory": "lib"
```

Comment:
Since this is removed, I think we need to now specify `"files"` in the package.json?